workflows/metatdenovo.nf

```diff
@@ -224,8 +227,18 @@ workflow METATDENOVO {
             UNPIGZ_MEGAHIT_CONTIGS.out.unzipped.collect { [ [ id: 'all_samples' ], it ] }
         )
         ch_gff = TRANSDECODER.out.gff.map { it[1] }
+        ch_hmmr_aa = TRANSDECODER.out.pep
     }
 
+    //
+    // MODULE: Hmmsearch on orf caller output
+    //
+    if( params.hmmsearch) {
+        ch_hmmrpath = Channel.fromPath(params.hmmpath)
```

Comment:
I assume this code only works if `hmmpath` has a value like `hmms/*.hmm`, i.e. a file glob. I think it might be better to separate `hmmpath` into `hmmdirectory` and `hmmpattern`, the latter with a default of `*.hmm`.